adminpages/orders.php

```diff
@@ -1711,13 +1711,14 @@ function pmpro_ShowMonthOrYear() {
 							<a href="<?php echo esc_url( add_query_arg( array( 'page' => 'pmpro-orders', 's' => $order->subscription_transaction_id ), admin_url( 'admin.php' ) ) ); ?>" title="<?php esc_attr_e( 'View all orders for this subscription', 'paid-memberships-pro' ); ?>" class="pmpro_order-renewal"><?php esc_html_e( 'Renewal', 'paid-memberships-pro' ); ?></a>
 						<?php } ?>
 					</td>
-					<td class="column-date">
+					<td class="
```

Comment:
This HTML class was accidentally removed.

Reply:
Other than that the PR looks good.

Reply:
Fixed by cfc7b14322a038fb16139fea189205e281001fb5. Sorry.

Reply:
No problem! These things happens :) moved this to ready for merge.